engine/src/flutter/impeller/entity/shaders/blending/porter_duff_blend.frag

```diff
@@ -46,9 +46,8 @@ void main() {
                        frag_info.tmy) *
                 frag_info.input_alpha;
   f16vec4 src = v_color;
-  frag_color =
-      src * (frag_info.src_coeff + dst.a * frag_info.src_coeff_dst_alpha) +
-      dst * (frag_info.dst_coeff + src.a * frag_info.dst_coeff_src_alpha +
-             src * frag_info.dst_coeff_src_color);
+  frag_color = f16vec4(src * (src_coeff + dst.a * src_coeff_dst_alpha) +
```

Comment:
nit; but just a nice documentation trail:
// see GetPorterDuffSpecConstants for actual constants

Reply:
Done